src/commands/LatexCommand.ts

```diff
@@ -0,0 +1,71 @@
+import { Command } from "../Command";
+import { CommandInteraction, MessageActionRow, MessageButton } from "discord.js";
+import { SlashCommandBuilder, SlashCommandSubcommandsOnlyBuilder } from "@discordjs/builders";
+import { unknownSubcommandEdit } from "../util/unknownSubcommand";
+import { latexEquation, latexMixed } from "../util/latexCommand/latexRendering";
+
+class Latex extends Command {
+	constructor() {
+		super("latex");
+	}
+
+	async execute(interaction: CommandInteraction): Promise<void> {
+		await interaction.deferReply();
+
+		const subcommand = interaction.options.getSubcommand();
+		const input = interaction.options.getString("input", true);
+
+		let urlToFile: string | null;
+		if (subcommand === "equation") {
+			urlToFile = await latexEquation(input);
+			this.answerWithImage(interaction, urlToFile, input);
+		} else if (subcommand === "inline") {
+			urlToFile = await latexMixed(input);
+			this.answerWithImage(interaction, urlToFile, input);
+		} else {
+			unknownSubcommandEdit(interaction);
+		}
+	}
+
+	answerWithImage(interaction: CommandInteraction, urlToFile: string | null = null, input: string): void {
+		const row = new MessageActionRow().addComponents(
+			new MessageButton()
+				.setCustomId(`latex.delete.${input}`)
```

Comment:
This will easily break if input goes above the character limits of custom ids. Also not encoded so a dot in input will break it too. I just suggest saying the compilation went wrong and try manually instead of button.

Reply:
You know you can just copy paste your old command?